governator-core/src/test/java/com/netflix/governator/package1/AutoBindSingletonMultiBinding.java

```diff
@@ -0,0 +1,11 @@
+package com.netflix.governator.package1;
+
+import com.netflix.governator.annotations.AutoBindSingleton;
+
+import javax.inject.Singleton;
+
+@AutoBindSingleton(multiple=true, value=AutoBindSingletonInterface.class)
+@Singleton
+public class AutoBindSingletonMultiBinding implements AutoBindSingletonInterface {
```

Comment:
seems redundant to force declaration of an autobindsingleton as a singleton.  What else would it be?